tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,824 @@
+            # Verify required keys
+            required_keys = [
+                "DRBD_DEVICE_NAME",
+                "DRBD_RESOURCE_NAME",
+                "NODE_0_NAME",
+                "NODE_1_NAME",
+            ]
+            for key in required_keys:
+                assert key in cm.get(
+                    "data", {}
+                ), f"Required key {key} not found in DRBD ConfigMap"
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)